manifests/g/gerardog/gsudo/2.1.0/gerardog.gsudo.installer.yaml

```diff
@@ -0,0 +1,30 @@
+# Created with WinGet Releaser v2 using Komac v1.8.2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: gerardog.gsudo
+PackageVersion: 2.1.0
+InstallerType: msi
+Scope: machine
+InstallerSwitches:
+  Silent: /quiet /norestart
+  SilentWithProgress: /passive /norestart
+UpgradeBehavior: install
+Commands:
+- gsudo
+- sudo
+ReleaseDate: 2023-07-09
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/gerardog/gsudo/releases/download/v2.1.0/gsudo.setup.x86.msi
+  InstallerSha256: 4BBC4ECC8ABA137958B2CF683E9627ABEE9DAACD265F98256D137051B34B40D7
+  ProductCode: '{3DBF0D3C-BA18-44CF-AE40-5C0AEA81C60F}'
+- Architecture: x64
+  InstallerUrl: https://github.com/gerardog/gsudo/releases/download/v2.1.0/gsudo.setup.x64.msi
+  InstallerSha256: 3BA8A3F79AA5408944797038EE57D93D1E331A77B960A2B0F3BB2E0B7A18D9E3
+  ProductCode: '{E918BCF9-EED2-465A-862C-F5BFFC25F82A}'

```

Comment:
```suggestion
  ProductCode: '{6296AF88-0AF9-4DDA-B285-88D7FC75E2C7}'
```

Reply:
Hi @vedantmgoyal2009 do you have any idea why winget releaser v2 may have used different ProductCodes this time?
https://github.com/gerardog/gsudo/actions/runs/5497391130/jobs/10018145060#step:7:1